node/pegnet/txrelation.go

```diff
@@ -0,0 +1,52 @@
+package pegnet
+
+import (
+	"database/sql"
+
+	"github.com/Factom-Asset-Tokens/factom"
+)
+
+// createTableTransactions is a SQL string that creates the
+// "pn_address_transactions" table.
+//
+// The "pn_address_transactions" table has a foreign key reference to the
+// "pn_addresses" table, which must exist first.
+//
+// If the transaction is a conversion, both "to" and "conversion" are set to true
+const createTableTransactions = `CREATE TABLE "pn_address_transactions" (
+        "entry_hash"    BLOB NOT NULL UNIQUE,
+        "address_id"    INTEGER NOT NULL,
+        "tx_index"      INTEGER NOT NULL,
+        "to"            BOOL NOT NULL,
+        "conversion"    BOOL NOT NULL,
+
+        PRIMARY KEY("entry_hash", "address_id"),
+
+        FOREIGN KEY("address_id") REFERENCES "pn_addresses"
+);
+CREATE INDEX "idx_address_transactions_address_id" ON "pn_address_transactions"("address_id");
```

Comment:
Am I correct in thinking this is just so we can query for all transactions related to an address?

Reply:
Yes, that is the reasoning in my mind too. Just copied what fatd is doing: https://github.com/Factom-Asset-Tokens/fatd/blob/develop/db/addresses/txrelation.go#L20

Not sure if we really need it, but I also don't think it hurts us or adds unneeded complexity

Reply:
Agreed, just was wondering if it was purely for indexing.